recipe/meta.yaml

```diff
@@ -38,6 +49,8 @@ test:
     - ${CXX} example_EventSelection.cc -o example_EventSelection -I${CONDA_PREFIX}/include -L${CONDA_PREFIX}/lib -lHepMC
     # example_BuildEventFromScratch
     - ${CXX} example_BuildEventFromScratch.cc -o example_BuildEventFromScratch -I${CONDA_PREFIX}/include -L${CONDA_PREFIX}/lib -lHepMC
+    # temporarily skipping pip check on win and pypy build because of bizarre tqdm issue
```

Comment:
Can you post the `tqdm` issue that you're seeing?

Reply:
Oh, sorry for a confusion. That was a copy-pasted code snippet when I was figuring out how testing works for a package. It have nothing to do with this particular package, I simply forgot to remove it.